packages/mdc-textfield/foundation.js

```diff
@@ -57,6 +57,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.inputBlurHandler_ = () => this.deactivateFocus_();
     this.inputInputHandler_ = () => this.autoCompleteFocus_();
     this.inputKeydownHandler_ = () => this.receivedUserInput_ = true;
+    this.useNativeValidityChecking_ = true;
```

Comment:
the word "native" here is a little misleading. Makes it sound like its happening in an Android/iOS app, not web.

I think what you mean is that MDC Textfield has some default validity checking, and this variable overrides that.

why not `useCustomValidityChecking_` and flip true to false?